clearwater-etcd.root/etc/logrotate.d/clearwater-etcd

```diff
@@ -0,0 +1,11 @@
+/var/log/clearwater-etcd/clearwater-etcd.log
```

Comment:
This file already exists - https://github.com/Metaswitch/clearwater-etcd/blob/master/clearwater-etcd/etc/logrotate.d/clearwater-etcd